### PR TITLE
organizing functions for testing surface code

### PR DESCRIPTION
**Pull request by juhuntenburg**
Moving helper functions in separate file

### CODE REVIEW

**Comment by KamalakerDadi** [comment]
Just flake8 issues. https://travis-ci.org/nilearn/nilearn/jobs/520799043#L437

Otherwise, LGTM.

**Comment by KamalakerDadi** [comment]
Could you tell us the motivation to move to separate file ?

**Comment by jeromedockes** [comment]
> Could you tell us the motivation to move to separate file ?

as far as I can tell these functions are only used in test_surface? why create a testing module ouside of the tests package?

**Comment by juhuntenburg** [comment]
Actually, I was asked by @SylvainLan and @GaelVaroquaux to do it this way. I don't have particular strong feelings about it.

File: nilearn/surface/testing.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Helper functions for testing surface code
+
+import numpy as np
+from scipy.spatial import Delaunay
+
+
+def _generate_surf():
```
Comment (GaelVaroquaux) [inline nilearn/surface/testing.py]: These functions should probably not be private (though the module might be). They should also have a tiny docstring.

**Comment by GaelVaroquaux** [comment]
The reason to move it is to foster reuse in the surface code (which is going to grow soon).

Sorry, @juhuntenburg, I'm asking you to add docstring, in other words, you are victim of a classic: you are asked to improve code just because you are moving it.

**Comment by jeromedockes** [comment]
and why move them outside of the tests package? can any of these functions be used for something else than testing? if so, the module should have a different name

**Comment by GaelVaroquaux** [comment]
> and why move them outside of the tests package?

Because only tests go in there, no utilities to test.

> can any of these functions be used for something else than testing? if so, the module should have a different name

The file is called "testing.py", which is a common name for testing
utilities.

**Comment by jeromedockes** [comment]
> Because only tests go in there, no utilities to test.

ok thanks!

**Comment by kchawla-pi** [comment]
> The file is called "testing.py", which is a common name for testing

Should we rename them to `testing_utils.py` for future contributors?

**Comment by GaelVaroquaux** [comment]
Fine with me 

⁣Sent from my phone. Please forgive typos and briefness.​

**Comment by GaelVaroquaux** [comment]
Great! Merging!

**Pull request merged**